geometry_msgs/PoseStamped[] involved_locations
geometry_msgs/PoseStamped current_attached_object_start_pose
geometry_msgs/PoseStamped future_attached_object_start_pose
geometry_msgs/PoseStamped future_placed_object_end_pose